api/tools/src/main/java/org/pmiops/workbench/tools/CreateTerraMethodSnapshot.java

```diff
@@ -65,6 +68,32 @@
 
   private static final Logger log = Logger.getLogger(CreateTerraMethodSnapshot.class.getName());
 
+  private static final Pattern importWdlRegex = Pattern.compile("import \"(.+)\" as (.+)");
+
+  private String resolveRelativeImports(String wdlSource, String rawGithubDirectoryUrl) {
```

Comment:
Probably a comment somewhere about Agora not being able to make relative references like this would be appropriate. It's a bit unclear just from reading the code as to why this step is necessary.